pom.xml

```diff
@@ -80,6 +80,7 @@
         <version.microservice.audit-api>1.3</version.microservice.audit-api>
         <version.microservice.authorization-api>1.3</version.microservice.authorization-api>
         <version.microservice.metrics-reporter>1.2</version.microservice.metrics-reporter>
+        <version.microservice.scanner-api>1.0-SNAPSHOT</version.microservice.scanner-api>
```

Comment:
There is a consideration about putting a snapshot version here. Currently, nothing in the "services" folder builds by default when you run a build from the top level (since the services folder is considered to effectively be a separate repository). Therefore, if a user were to pull and build, their build would fail. It works with released versions since we push those to our "repo" branch and the parent pom can pull from them. Eventually, of course, we'll be able to push those releases to maven central.